### ISSUE TITLE
Setting `existing: true` on an s3 lambda event causes deployment to fail

### ISSUE THREAD

**Issue by Ellosaur** [serverless/serverless#6638]
# This is a Bug Report

## Description

- What went wrong?
Adding the `existing: true` property to the function below causes the previously passing deployment to fail

- What did you expect should have happened?
Serverless should deploy, adding an s3 trigger event to the existing bucket that triggers the lambda function

- What was the config you used?
```yaml
...
functions:
  trigger-update:
    handler: bin/triggerUpdate/triggerUpdate_lambda
    reservedConcurrency: 1
    events:
      - s3:
          bucket: ${self:provider.existingBucket}
          event: s3:ObjectCreated:*
          existing: true # commenting out this line fixes the error but means a bucket is created
```

- What stacktrace or error message from your provider did you see?
```
Serverless: Installing dependencies for custom CloudFormation resources...

  Error --------------------------------------------------

  Error: Command failed: npm install --prefix C:\Users\ELLIOT~1.RIC\AppData\Local\Temp\tmpdirs-serverless\e175\29097f0ffcadbd41\resources
  npm ERR! code ENOLOCAL
  npm ERR! Could not install from "" as it does not contain a package.json file.

  npm ERR! A complete log of this run can be found in:
  npm ERR!     C:\Users\elliot.rice\AppData\Roaming\npm-cache\_logs\2019-09-05T15_25_23_483Z-debug.log

      at ChildProcess.exithandler (child_process.js:294:12)
      at ChildProcess.emit (events.js:188:13)
      at ChildProcess.EventEmitter.emit (domain.js:441:20)
      at maybeClose (internal/child_process.js:978:16)
      at Process.ChildProcess._handle.onexit (internal/child_process.js:265:5)
```

## Additional Data

```
  Your Environment Information ---------------------------
     Operating System:          win32
     Node Version:              11.6.0
     Framework Version:         1.51.0
     Plugin Version:            1.3.9
     SDK Version:               2.1.0
```

**Comment by abhishekdixit1508** [comment]
Hi, I was facing the same issue yesterday after serverless version upgrade. After spending a day :( I have downgraded the version to 1.49.0 (2019-07-30) and it just worked :). Hope serverless will fix this bug soon.

**Comment by Ellosaur** [comment]
I get the same error with the config below :-( 
```
  Your Environment Information ---------------------------
     Operating System:          win32
     Node Version:              11.6.0
     Serverless Version:        1.49.0
     Enterprise Plugin Version: 1.3.11
     Platform SDK Version:      2.1.1
```

**Comment by Ellosaur** [comment]
This appears fixed in version `1.53`
Possibly https://github.com/serverless/serverless/pull/6742 that fixes it.

Thanks!

**Comment by exoego** [comment]
@Ellosaur 
Thanks for reporting.
Closing since this fixed in `1.53.x`

**Comment by nicgirault** [comment]
Still not fixed in 1.65.0. Any plan to fix this?

**Comment by tysoncadenhead** [comment]
I'm seeing a similar issue with `existing: true` 

It gets to:
"Installing dependencies for custom CloudFormation resources"

and then just exists without deploying and without throwing any errors. If I remove `existing: true` from my s3 bucket, it continues to deploy
